app/styles/components/_horizontal-nav.scss

```diff
@@ -83,6 +83,10 @@
   width: -moz-max-content;
 }
 
+.live-nyc-txt {
+  width: -ms-max-content; //IE11 
```

Comment:
I _think_ you can include this in the `.live-nyc-text` css block just above this

Reply:
@bkammin For some reason, this would mess up the firefox.